package/MDAnalysis/core/__init__.py

```diff
@@ -390,22 +390,21 @@ def __doc__(self):
 
            >>> flags['%(name)s'] = value
 
-           The Bio.PDB reader (value=``False``) can deal with 'proper' PDB
-           files from the Protein Databank that contain special PDB features
-           such as insertion codes and it can auto-correct some common
-           mistakes; see :mod:`Bio.PDB` for details. However, Bio.PDB has been
-           known to read some simulation system PDB files **incompletely**; a
-           sure sign of problems is a warning that an atom has appeared twice
-           in a residue.
-
-           Therefore, the default for the PDB reader is ``True``, which
-           selects the "primitive" (or "permissive") reader
-           :class:`MDAnalysis.coordinates.PDB.PrimitivePDBReader`, which
+           The default for the PDB reader is ``True``, which
```

Comment:
I appreciate the extensive docs but in this case just remove all of this; users should not need to know about `permissive` anymore.

Just say:

This flag is not used anymore and will be removed in 0.16.0.